src/views/outbound/OutboundRequest.jsx

```diff
@@ -482,15 +495,42 @@ class OutboundRequest extends React.Component {
 
   handleSendSingleTestCase = async (testCaseIndex) => {
     const { test_cases, ...remainingProps } = this.state.template
-    const testCaseToSend = { test_cases: [ test_cases[testCaseIndex] ], ...remainingProps }
+    const testCaseToSend = { test_cases: [test_cases[testCaseIndex]], ...remainingProps }
     this.handleSendTemplate(testCaseToSend)
   }
 
+  toggleServerLogs = async (componentName=null) => {
+    let logs = componentName === 'TestCaseEditor' ? this.state.testCaseEditorLogs : this.state.logs
+    if (logs.length) {
+      componentName === 'TestCaseEditor' ? this.setState({ testCaseEditorLogs: [] }) : this.setState({ logs: [] })
+      return;
+    }
+    if (!this.state.lastOutgoingRequestID) return;
+    const res = await axios.get(`${getConfig().apiBaseUrl}/api/logs/search?metadata.trace.traceId=${this.state.lastOutgoingRequestID}`)
```

Comment:
If we want to change the endpoint in the TTK backend service, please handle that here